src/Root.tsx

```diff
@@ -0,0 +1,26 @@
+import {
+  Navigate,
+  Route,
+  HashRouter as Router,
+  Routes,
+} from 'react-router-dom';
+import { App } from './App';
+import { HomePage } from './pages/HomePage';
+import { PeoplePage } from './pages/PeoplePage';
+
+export const Root = () => (
+  <Router>
+    <Routes>
+      <Route path="/" element={<App />}>
+        <Route index element={<HomePage />} />
+        <Route path="home" element={<Navigate to="/" replace />} />
+        <Route path="/people">
+          <Route index element={<PeoplePage />} />
+          {/* <Route path=":slug" element={<PeoplePage />} /> */}
+          <Route path="/people/:slug" element={<PeoplePage />} />
```

Comment:
The path for the route should be ':slug' instead of '/people/:slug'. Since this is a nested route under '/people', it should not start with a leading slash.